WooFoundation/WooFoundation/Colors/WooFoundationBundleClass.swift

```diff
@@ -0,0 +1,3 @@
+/// Disposable Class to find this `Bundle` at runtime
+///
+internal class WooFoundationBundleClass {}
```

Comment:
I extracted this class from the UIColor+ColorStudio because some of the constructors used in it are not available in watchOS.